src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -594,6 +651,63 @@ private static int GetIntQueryParameterValue(NameValueCollection queryStringName
             return request.CreateResponse(HttpStatusCode.Accepted);
         }
 
+        private async Task<HttpResponseMessage> HandleGetEntityRequestAsync(
+            HttpRequestMessage request,
+            EntityId entityId)
+        {
+            IDurableOrchestrationClient client = this.GetClient(request);
+
+            var response = await client.ReadEntityStateAsync<JToken>(entityId);
+
+            if (!response.EntityExists)
+            {
+                return request.CreateResponse(HttpStatusCode.NotFound);
+            }
+            else
+            {
+                return request.CreateResponse(HttpStatusCode.OK, response.EntityState);
+            }
+        }
+
+        private async Task<HttpResponseMessage> HandlePostEntityOperationRequestAsync(
+            HttpRequestMessage request,
+            EntityId entityId)
+        {
+            IDurableOrchestrationClient client = this.GetClient(request);
+
+            string operationName = request.GetQueryNameValuePairs()["op"] ?? string.Empty;
+
+            if (request.Content == null)
+            {
+                await client.SignalEntityAsync(entityId, operationName);

```

Comment:
What does this do for a non-existent entity?

Reply:
entities get created automatically when they receive a signal or call.